apps/scratchpad/src/index.ts

```diff
@@ -43,11 +51,74 @@ function tsDesc() {
 async function ts() {
   console.log("Hello, world");
 
-  const num =
-    23658957077183673623975163142370986705703709099221578179586031616n;
-  console.log(`Hex length: ` + num.toString(16).length);
-  console.log(`toHex() length: ` + numberToHex(num).length);
-  console.log(`toHex(64) length: ` + numberToHex(num, { size: 64 }).length);
+  const publicClient = createPublicClient({ chain: base, transport: http() });
```

Comment:
dumb q, shouldn't this be `baseGoerli`?

Reply:
ah I see, this is only used for the call for the fee estimation question we were wondering about yesterday. recommend putting a comment or moving to its own function for clarity.

Reply:
yes this is just the scratchpad, will rename